Sources/Sentry/SentryHttpTransport.m

```diff
@@ -52,7 +52,7 @@ - (void)    sendEvent:(SentryEvent *)event
     
     NSError *requestError = nil;
     // TODO: We do multiple serializations here, we can improve this
-    SentryNSURLRequest *request = [[SentryNSURLRequest alloc] initStoreRequestWithDsn:self.options.dsn
+    NSURLRequest *request = [[SentryNSURLRequest alloc] initStoreRequestWithDsn:self.options.dsn
```

Comment:
Why this change?

Reply:
Because of the [principle of least knowledge](https://en.wikipedia.org/wiki/Law_of_Demeter). `sendRequest` doesn't need to know it is a `SentryNSURLRequest`. Only knowing about `NSURLRequest` it is more flexible.